README.md

```diff
@@ -8,7 +8,7 @@ Skip this if you prefer to pull from [docker hub](https://hub.docker.com/u/elron
 
 ```bash
 cd mainnet
-docker image build . -t elrondnetwork/elrond-node-obs:v1.3.50 -f ./elrond-node-obs
+docker image build . -t elrondnetwork/elrond-node-obs:v1.3.50-rc1 -f ./elrond-node-obs
```

Comment:
we need the exact version v1.3.50 not the `-rc1` one

Reply:
fixed